### DISCUSSION TITLE
[Feature] Add the ability to launch a slideshow for places and search results

### DISCUSSION THREAD

**Discussion by imler** [immich-app/immich#8711] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

If I’m correct it’s impossible to launch a slideshow for places and search results right now. 

It would be great to have that function, so I could see a slideshow of all photos taken in a selected town for example.

As far as I understand, places are technically search results so it would also apply to them 

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by solluh** [comment]
I would like this too, as another example: 
the option to select the display of images according to star rating; I cannot expect some viewers to see all the pictures in my album and would only like to display, for example, the photos with 3 to 5-star ratings.
